packages/oraidex-server/src/helper.ts

```diff
@@ -56,3 +71,55 @@ export const getOrderbookTicker = async () => {
     return [];
   }
 };
+
+// fetch the simulate prices
+export const fetchSimulatePrices = async () => {
+  const cosmwasmClient = await CosmWasmClient.connect(rpcUrl);
+  const routerContract = new OraiswapRouterQueryClient(
+    cosmwasmClient,
+    process.env.ROUTER_CONTRACT_ADDRESS || "orai1j0r67r9k8t34pnhy00x3ftuxuwg0r6r4p8p6rrc8az0ednzr8y9s3sj2sf"
```

Comment:
lets use the constant router contract from oraidex common

Reply:
Resolve